tests/00-setup

```diff
@@ -36,9 +36,10 @@ createvenv() {
         [ $updated_downloadcache -eq 0 ] && bzr up $DOWNLOADCACHE
         virtualenv --distribute --extra-search-dir=$DOWNLOADCACHE $VENV
         . $ACTIVATE && \
-            yes w | env BZR_PLUGIN_PATH='-user' \
-            pip install --find-links releases --find-links jujugui-deps jujugui
-            pip install -r $TEST_REQUIREMENTS --find-links releases --find-links deps --find-links $DOWNLOADCACHE --no-allow-external --no-index
+        pip install --upgrade pip && \
```

Comment:
I was aware it was the venv one; it didn't occur to me that the venv one wouldn't be the latest. I'm fine with this as is then, thanks.